pkg/controllers/migration/virtualmachine.go

```diff
@@ -769,5 +797,49 @@ func (h *virtualMachineHandler) checkAndCreateVirtualMachineImage(vm *migration.
 		}
 	}
 
-	return h.vmi.Create(vmi)
+	vmiObj, err := h.vmi.Create(vmi)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create Kubevirt VMI (namespace=%s spec.displayName=%s): %v", vmi.Namespace, vmi.Spec.DisplayName, err)
```

Comment:
minor tweak to the error message since kubevirt vmi is a different object.

```suggestion
		return nil, fmt.Errorf("failed to create Harvester VirtualMachineImage (namespace=%s spec.displayName=%s): %v", vmi.Namespace, vmi.Spec.DisplayName, err)
```